pkg/openwsn/include/board_info.h

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (C) 2017 Hamburg University of Applied Sciences
```

Comment:
(copyright accurate?)

Reply:
The OpenWSN port was started by @PeterKietzmann so that is why I kept the original copyright, I guess I should add Inria as well.

Reply:
yup